Cython/Compiler/Nodes.py

```diff
@@ -4709,6 +4676,31 @@ def analyse_declarations(self, env):
             else:
                 scope.implemented = 1
 
+        if len(self.bases.args) > 1:
```

Comment:
Looks like you changed the indentation space in a couple of places (4->2). I would prefer sticking with the standard of 4 spaces everywhere.

Reply:
Not intentionally, thanks for catching.